tests/python/pants_test/engine/legacy/test_graph.py

```diff
@@ -78,7 +78,7 @@ def test_invalidate_fsnode_incremental(self):
 
       # Invalidate the '3rdparty/python' DirectoryListing, the `3rdparty` DirectoryListing,
       # and then the root DirectoryListing by "touching" files/dirs.
-      for filename in ('3rdparty/python/BUILD', '3rdparty/python', 'non_existing_file'):
+      for filename in ('3rdparty/python/BUILD', '3rdparty/python', ''):
```

Comment:
any particular reason for this change? does the literal `non_existing_file` path cause issues with the new impl or something?